prac_06/languages.py

```diff
@@ -0,0 +1,21 @@
+from programming_language import ProgrammingLanguage
```

Comment:
The client code demonstrates how to create instances of ProgrammingLanguage and display dynamically typed languages.